src/set_theory/ordinal_arithmetic.lean

```diff
@@ -1861,6 +1861,9 @@ theorem omega_le {o : ordinal.{u}} : omega ≤ o ↔ ∀ n : ℕ, (n : ordinal)
    let ⟨n, e⟩ := lt_omega.1 h in
    by rw [e, ← succ_le]; exact H (n+1)⟩
 
+theorem omega_eq_sup_nat_cast : omega = sup nat.cast :=
```

Comment:
reverse equality (and rename)

Reply:
I've reversed the equalities and simplified the theorem names while at it.